easybuild/easyconfigs/s/STEAK/STEAK-master-foss-2019b.eb

```diff
@@ -0,0 +1,40 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+# You will need to download the tarball manually and put it in the folder
+# where this file is. 
+
+easyblock = 'MakeCp'
+
+name = 'STEAK'
+version = 'master'
+
+homepage = 'https://omictools.com/steak-tool'
+description = """Detects integrations of any sort in high-throughput 
+sequencing (HTS) data. STEAK was built for validating and discovering 
+transposable element (TE) and retroviral integrations in a variety of 
+HTS data. The software performs on both single-end (SE) and paired-end (
+PE) libraries and on a variety of HTS sequencing strategies. It can 
+be applied to a broad range of research interests and clinical uses 
+such as population genetic studies and detecting polymorphic integrations."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+sources = ['steak.zip']
```

Comment:
Add version to this filename, else it will collide when multiple versions have been downloaded. Add a comment to indicate which git hash this download had.

Reply:
As above.